### DISCUSSION TITLE
Kan man slette en gruppe?

### DISCUSSION THREAD

**Discussion by MartinQwist** [xooom-community/public#36] [Questions] [answered]
Et par arrangører i Kbh har spurgt, hvordan man sletter grupper. Jeg kan ikke rigtig se, at man kan. 

Er der en måde? 

Hvis ikke, så skal det tilføjes i Volunteering projektet, at arrangører skal have mulighed for at slette grupper. Eller evt. deaktivere og skjule dem, hvis det giver bedre mening ift. udvikling, arkitektur eller data.

**Comment by mikkelsl** [comment, marked as answer]
Nej, det kan man ikke. Du må meget gerne prioritere den i Volunteering.
